addons/website/static/src/js/editor/snippets.options.js

```diff
@@ -1878,30 +1876,32 @@ options.registry.collapse = options.Class.extend({
      * @private
      */
     _createIDs: function () {
-        var time = new Date().getTime();
-        var $tab = this.$target.find('[data-toggle="collapse"]');
+        let time = new Date().getTime();
+        const $tab = this.$target.find('[data-toggle="collapse"]');
+        const $panel = this.$target.find('.collapse');
+        const $tablist = this.$target.closest('.accordion');
 
         // link to the parent group
-        var $tablist = this.$target.closest('.accordion');
-        var tablist_id = $tablist.attr('id');
-        if (!tablist_id) {
-            tablist_id = 'myCollapse' + time;
-            $tablist.attr('id', tablist_id);
+        let tablistId = $tablist.attr('id');
+        if (!tablistId || $('[id="' + tablistId + '"]').length > 1) {
+            while ($('#' + (tablistId = 'myCollapse' + time)).length) {
+                time++;
+            }
+            $tablist.attr('id', tablistId);
```

Comment:
Is all this fixing the data-parent bug we talked about? Should not it be in stable versions 14.0 or even before?